### PR TITLE
fix styling + zem/mstone/xstone/ancient pieces

### PR DESCRIPTION
**Pull request by ocambridge**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "lodash": "^4.17.5",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "react-router": "^4.2.0",
```
Comment (ocambridge) [inline package.json:9]: had to add this to make ```yarn start``` work?

File: public/index.html [hunk 2]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```
Comment (ocambridge) [inline public/index.html]: this line should probably be removed now ive added the mobile version below

File: src/App.css [hunk 3]
```diff
@@ -47,3 +47,19 @@ input {
 p {
   font-size: 1rem;
 }
+
+/* mobile */ @media only screen
+and (min-device-width : 320px) and (max-device-width : 667px) {
+
+  .App-search,
+  .App-data {
+    max-width: 250px;
```
Comment (ocambridge) [inline src/App.css:56]: stops display pushing out on mobile because of padding

File: package.json (hunk 1 above)
Comment (samcambridge) [reply package.json:9]: weird, but no problem we'll need it when we add the stat calculator and stuff

File: public/index.html (hunk 2 above)
Comment (samcambridge) [reply public/index.html]: yeah

Comment (samcambridge) [reply public/index.html]: take it away

**Review by samcambridge — COMMENTED** [review]
Remove that viewport you mentioned and all good to go!

**Review by samcambridge — APPROVED** [review]

**Pull request merged**